client/header/activity-panel/panels/orders.js

```diff
@@ -35,173 +36,236 @@ import ActivityOutboundLink from '../activity-outbound-link';
+						>
+							{ __( 'Begin fulfillment' ) }
+						</Button>
+					}
+				>
+					<OrderStatus order={ order } />
+				</ActivityCard>
+			);
+		} );
 		return (
 			<Fragment>
-				<EmptyContent
-					title={ title }
-					actionLabel={ actionLabel }
-					actionURL={ null }
-					actionCallback={ actionCallback }
-				/>
+				{ cards }
+				<ActivityOutboundLink href={ 'edit.php?post_type=shop_order' }>
+					{ __( 'Manage all orders', 'woocommerce-admin' ) }
+				</ActivityOutboundLink>
 			</Fragment>
 		);
 	}
 
-	const menu = (
-		<EllipsisMenu label="Demo Menu">
-			<MenuTitle>Test</MenuTitle>
-			<MenuItem onInvoke={ noop }>Test</MenuItem>
-		</EllipsisMenu>
-	);
+	render() {
+		const { orders, isRequesting, isError, orderStatuses } = this.props;
+
+		if ( isError ) {
+			if ( ! orderStatuses.length ) {
+				return (
+					<EmptyContent
+						title={ __(
+							"You currently don't have any actionable statuses. " +
+								'To display orders here, select orders that require further review in settings.',
+							'woocommerce-admin'
+						) }
+						actionLabel={ __( 'Settings', 'woocommerce-admin' ) }
+						actionURL={ getAdminLink( 'admin.php?page=wc-admin#/analytics/settings' ) }
+					/>
+				);
+			}
 
-	const getCustomerString = order => {
-		const extended_info = order.extended_info || {};
-		const { first_name, last_name } = extended_info.customer || {};
+			const title = __(
+				'There was an error getting your orders. Please try again.',
+				'woocommerce-admin'
+			);
+			const actionLabel = __( 'Reload', 'woocommerce-admin' );
+			const actionCallback = () => {
+				// @todo Add tracking for how often an error is displayed, and the reload action is clicked.
+				window.location.reload();
+			};
 
-		if ( ! first_name && ! last_name ) {
-			return '';
+			return (
+				<Fragment>
+					<EmptyContent
+						title={ title }
+						actionLabel={ actionLabel }
+						actionURL={ null }
+						actionCallback={ actionCallback }
+					/>
+				</Fragment>
+			);
 		}
 
-		const name = [ first_name, last_name ].join( ' ' );
-		return sprintf(
-			__(
-				/* translators: describes who placed an order, e.g. Order #123 placed by John Doe */
-				'placed by {{customerLink}}%(customerName)s{{/customerLink}}',
-				'woocommerce-admin'
-			),
-			{
-				customerName: name,
-			}
+		const menu = (
+			<EllipsisMenu label="Demo Menu">
+				<MenuTitle>Test</MenuTitle>
+				<MenuItem onInvoke={ noop }>Test</MenuItem>
+			</EllipsisMenu>
 		);
-	};
-
-	const orderCardTitle = order => {
-		const { extended_info, order_id } = order;
-		const { customer } = extended_info || {};
-		const customerUrl = customer.customer_id
-			? getNewPath( {}, '/analytics/customers', {
-					filter: 'single_customer',
-					customers: customer.customer_id,
-				} )
-			: null;
+
+		const title =
+			isRequesting || orders.length
+				? __( 'Orders', 'woocommerce-admin' )
+				: __( 'No orders to ship', 'woocommerce-admin' );
```

Comment:
Should this be "No orders to fulfill" to match other language?

Reply:
According to the designs in p6riRB-4kT-p2, it should be "No orders to ship". @josemarques should we change this?